scss/_variables.scss

```diff
@@ -492,26 +492,26 @@ $table-cell-padding-x-sm:     .25rem !default;
 
 $table-cell-vertical-align:   top !default;
 
-$table-color:                 $body-color !default;
-$table-bg:                    transparent !default;
+$table-color:                 var(--bs-table-color) !default;
+$table-bg:                    var(--bs-table-bg) !default;
```

Comment:
In SCSS variables list we set reference to a CSS variable, possible here to override on application side:
* specific value, e.g. `$table-bg: $red;`
* reference to another CSS variable `$table-bg: var(--primary-color);`
* set to null `$table-bg: null;`
* set to custom expression of CSS other variables `$table: hsl(var(--color),var(--l));`